test/utils/dom/test_scrollable.ts

```
@@ -0,0 +1,89 @@
+// findScrollableChild touches DOM APIs (querySelectorAll, scrollHeight,
+// clientHeight, getComputedStyle). Rather than spin up jsdom, we mock
+// just enough of the HTMLElement interface to drive the function.
+
+interface MockElement {
+  scrollHeight: number;
+  clientHeight: number;
+  overflowY?: string;
+  overflow?: string;
+}
+
+function fakeContainer(children: MockElement[]): HTMLElement {
```

ESLint

[error] 'HTMLElement' is not defined.

(no-undef)